sdkconfig.defaults

```diff
@@ -7,14 +25,14 @@
 #
 # Serial flasher config
 #
-#CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
-#CONFIG_ESPTOOLPY_BAUD_115200B=y
+CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
+# CONFIG_ESPTOOLPY_BAUD_115200B is not set
 # CONFIG_ESPTOOLPY_BAUD_230400B is not set
-# CONFIG_ESPTOOLPY_BAUD_921600B is not set
+CONFIG_ESPTOOLPY_BAUD_921600B=y
```

Comment:
The upstream esp-idf didn't change their default, so this is a behaviour change in this PR. We can change these defaults, but then we should also mention these changes in this change request.